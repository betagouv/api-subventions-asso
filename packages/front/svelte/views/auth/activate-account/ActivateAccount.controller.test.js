import ActivateAccountController from "./ActivateAccount.controller";

describe("ActivateAccountController", () => {
    let controller;

    afterEach(() => (controller = null));

    describe("constructor", () => {
        beforeEach(() => (controller = new ActivateAccountController()));
        it("should define form steps", () => {
            const actual = controller.steps;
            expect(actual).toMatchSnapshot();
        });
    });
});
