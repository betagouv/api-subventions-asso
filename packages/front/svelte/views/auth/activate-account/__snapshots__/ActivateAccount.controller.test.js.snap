// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActivateAccountController constructor should define form steps 1`] = `
[
  {
    "alert": [Function],
    "component": [Function],
    "name": "DÃ©finir un mot de passe",
  },
]
`;
